docs/processes/release/tasks/publish/release.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Publish - Version 76.1 - WIP
```

Comment:
I find the filename and the Markdown page title confusing, as well as the parent directory being called "publish". Is there a more specific name to help distinguish what this page represents, specifically?

Reply:
This is in the same folder and uses the same header as the `index.md` file.
And that is because this file replaces big chunks of the `index.md` file.

Reply:
Okay, this is a WIP anyways. Maybe after this gets merged, we can consider renaming the title (in the Markdown "frontmatter" section) as a sufficient resolution. Let's skip this for now.